sentences/nl/light_HassLightSet.yaml

```diff
@@ -1,4 +1,23 @@
 language: nl
 intents:
   HassLightSet:
-    data: []
+    data:
+      # brightness
+      - sentences:
+          - "zet <name> op <brigtness> procent"
```

Comment:
You are referring to expansion rule `<brigtness>` (typo btw, missing h).

This expansion rule is not defined in `sentences/nl/_common.yaml`. Take a look at how English does it:

 - specify a list to define valid values of brightness (0…100)
 - specify an expansion rule that catches a list item, then renames it to `brightness_pct`, which is the valid slot name for the HassLightSet intent

https://github.com/home-assistant/intents/blob/9004373ef6350f4829a30c8ecfeb0e14da9057b3/intents.yaml#L61-L63

Reply:
Ah, that was the missing link :) I will update it